content/docs/projects/trust-manager.md

```diff
@@ -3,44 +3,58 @@ title: trust-manager
 description: 'Distributing Trust Bundles in Kubernetes'
 ---
 
-## Distributing Trust Bundles in Kubernetes
+trust-manager is the easiest way to manage TLS trust bundles in Kubernetes and OpenShift clusters.
 
-trust-manager is an operator for distributing trust bundles across a Kubernetes cluster.
-trust-manager is designed to complement
-[cert-manager](https://github.com/cert-manager/cert-manager) by enabling services to
-trust X.509 certificates signed by Issuers, as well as external CAs which may
-not be known to cert-manager at all.
+## Overview
+
+trust-manager is a small Kubernetes operator which aims to help reduce the overhead of managing
+TLS trust bundles in your clusters.
+
+It adds the `Bundle` custom Kubernetes resource (CRD) which can read input from various sources
+and combine the resultant certificates into a bundle ready to be used by your applications.
+
+trust-manager ensures that it's both quick and easy to keep your trusted certificates up-to-date
+and enables cluster administrators to easily automate providing a secure bundle without having
+to worry about rebuilding containers to update trust stores.
+
+It's designed to complement cert-manager and works well when consuming CA certificates from a
+cert-manager `Issuer` or `ClusterIssuer` but can be used entirely independently from cert-manager
+if needed.
 
 ## Usage
 
-trust ships with a single cluster scoped `Bundle` resource. A Bundle represents
-a set of data that should be distributed and made available across the cluster.
-There are no constraints on what data can be distributed.
+trust-manager ships with a single cluster scoped `Bundle` resource. A `Bundle` represents
```

Comment:
`a single cluster scoped Bundle resource` -> this confuses me. you can have multiple resources, it just ships with one CRD or one resource type right?

Reply:
Agree, the wording is clumsy and confusing and needs more than just the update from trust -> trust-manager. I'll change.